README.md

```diff
@@ -19,7 +18,7 @@ add the following line to your code:
 
     dc.set_api_key('YOUR-API-KEY')
```

Comment:
Should we just remove the note about the api key? We've removed it from the rest of our documentation (and the link on line 16 is a 404..)

Reply:
Removed API key section